### DISCUSSION TITLE
ru/start/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#711] [Announcements]
# ru/start/

Каталог
Присоединение к серверу (join.md): Как присоединиться к нашему серверу и начать играть.; 
Верификация (verify.md): Процесс верификации для обеспечения безопасности на се...

https://korepi.com/ru/start/

**Comment by DeMoH228** [comment]
помогите пж
у меня аккаунт в дс взломали и вышли с дс сервера вашего(

**Reply by QweRezOn** [reply]
Свяжись со мной в дискорде. qwerezon

**Comment by potop1** [comment]
Добрый день, не могу понять, как правильно установить чит, помогите
